### PR TITLE
Marketplace tutorial

### PR DESCRIPTION
**Pull request by ccondra**
## Purpose of this pull request

This pull request (PR)i is an  example for the Marketplace tutorial.

## Affected DevDocs pages

-  [Company Profile](https://devdocs.magento.com/marketplace/sellers/profile-company.html)
-  marketplace-sellers.yml

### CODE REVIEW

File: src/marketplace/sellers/account-setup.md [hunk 1]
```diff
@@ -6,6 +6,39 @@ redirect_from: /marketplace/sellers/account-setup-overview.html
...
+## Absolute links {#absolute-links}
+
+-  [Process overview](https://devdocs.magento.com/marketplace/sellers/account-setup-process.html)
+-  [Register as a developer](https://devdocs.magento.com/marketplace/sellers/developer-register.html)
+-  [Marketplace developer portal](https://devdocs.magento.com/marketplace/sellers/developer-portal.html)
+
+## Reference Links {#reference-links}
+
+
+Lorem ipsum dolor sit amet, [process overview] sed cibo graeco inermis ex. Doctus labitur assueverit ut quo, ei vero adipisci sit. Accusata partiendo eloquentiam no nam, ad duo graeci splendide. Mei dicta everti feugait eu, [register] putent vidisse dolorum has ea. Mea odio ipsum malorum cu, sit cu cetero interesset, et usu sumo cetero intellegam. Mei ceteros splendide appellantur id, [portal] et intellegat temporibus eam.
```
Comment (dobooth) [inline src/marketplace/sellers/account-setup.md:30]: Can this be done in English?

Comment (BarnyShergold) [reply src/marketplace/sellers/account-setup.md:30]: That's Lorem Ipsum - dummy text - not sure why it's been added @ccondra?

File: src/marketplace/sellers/account-setup.md [hunk 2]
```diff
@@ -6,6 +6,39 @@ redirect_from: /marketplace/sellers/account-setup-overview.html
 
 Create a new Marketplace account, or use your existing Magento account. Then, use the Marketplace Developer Portal to manage your profile information, product submissions, and the monthly payments you receive from sales on Magento Marketplace.
 
+## Bulleted list {#bulleted-list}
```
Comment (BarnyShergold) [inline src/marketplace/sellers/account-setup.md:9]: Why has bulleted list been repeated 4 times with the headings as shown?

File: src/marketplace/sellers/profile-company.md [hunk 3]
```diff
@@ -22,6 +22,10 @@ Prepare an image for your company profile that is 255 pixels square. Then, save
    ![Company background]({{ site.baseurl }}/marketplace/sellers/images/account-company-profile-name.png){: .zoom}
    _Company Background_
 
+   -  Indented bullet
```
Comment (BarnyShergold) [inline src/marketplace/sellers/profile-company.md:25]: These extra bullets should be removed.

File: src/marketplace/sellers/profile-company.md [hunk 4]
```diff
@@ -75,7 +79,7 @@ Enter the link to each professional social network that you want to include in y
 ![Social networks]({{ site.baseurl }}/marketplace/sellers/images/account-profile-social-networks.png){: .zoom}
 _Social Networks_
 
-## Step 6: Activate your account
+## Step 5: Activate your account
```
Comment (BarnyShergold) [inline src/marketplace/sellers/profile-company.md:82]: Page now has two step 5s

**Review by BarnyShergold — CHANGES_REQUESTED** [review]
Pages seem to contain a lot of placeholders - is this PR complete or should it be "Work In Progress"?

File: src/marketplace/sellers/account-setup.md (hunk 1 above)
Comment (BarnyShergold) [reply src/marketplace/sellers/account-setup.md:30]: Blank lines are causing tests to fail

File: src/marketplace/sellers/account-setup.md (hunk 2 above)
Comment (ccondra) [reply src/marketplace/sellers/account-setup.md:9]: Sorry for the confusion -- please ignore this PR -- it's an example for a tutorial, and I needed to take screenshots.

**Comment by ccondra** [comment]
This was an example for a tutorial in progress.

**Pull request closed without merging**